sufia-models/lib/sufia/models/stats/user_stat_importer.rb

```diff
@@ -38,8 +38,12 @@ def date_since_last_cache(user)
       end
     end
 
-    def files_for_user(user)
-      ::GenericFile.where(Solrizer.solr_name('depositor', :symbol) => user.user_key)
+    def file_ids_for_user(user)
+      ids = []
+      ::GenericFile.find_in_batches("#{Solrizer.solr_name('depositor', :symbol)}:\"#{user.user_key}\"",{fq:"#{Solrizer.solr_name("has_model", :symbol)}:\"#{::GenericFile.to_class_uri}\"",fl:["id", "#{Solrizer.solr_name('depositor', :symbol)}"]}) do |group|
+        ids.concat group.map{|doc| doc["id"]}
```

Comment:
Oops! I confused #concat for #push. No, definitely stick with #concat instead of the shovel operator.  Spacing changes would be icing on the cake.  Thx!